### PR TITLE
Email validation and suggestion

### PR DESCRIPTION
**Pull request by hsztan**
My partner and I 👨🏻‍💻🧑‍💻 paired programmed functionality to my portfolio's email 📧 input form element to validate and check if the email is in lower-case letters. 🔡
If the email 📧 has uppercase-letters, it will also return a suggestion with a warning message ☢️.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -223,6 +223,7 @@ <h2 class="intro">
             placeholder="Email address"
             required
           />
+          <small id="email-warning"></small>
```
Comment (keroloussamy) [inline index.html]: - [x] You did a really great work the only missing part is as the requirements say the error messages should be at the bottom near the submit button.
` you show an error message to the user near the submit button informing them of the error and the form is not sent.`

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (hsztan) [reply index.html]: Thanks for the feedback 😎

**Review by thecodechaser — APPROVED** [review]
## Hi @hsztan 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**